### PR TITLE
[v6r14] Fix CPU normalization and few changes in RemoveReplica

### PR DESCRIPTION
**Pull request by phicharp**
RemoveReplica:
* Some cosmetics
* bulkRemoval() was modifying its input dict argument and returning it: useless, only modify argument

CPUNormalization:
* Get HS06 from JOBFEATURES if present

### CODE REVIEW

**Comment by chaen** [comment]
From the pure logic point of view, review OK, however landscape found new small style errors which I would encourage to correct (number of indent space and things like this)

File: DataManagementSystem/Agent/RequestOperations/RemoveReplica.py [hunk 1]
```diff
@@ -132,29 +127,27 @@ def bulkRemoval( self, toRemoveDict, targetSE ):
...
-    for lfn, opFile in toRemoveDict.items():
+    for lfn, opFile in toRemoveDict.iteritems():
       if lfn in removeReplicas["Failed"]:
         opFile.Error = str( removeReplicas["Failed"][lfn] )
-    return S_OK( toRemoveDict )
+    return S_OK()
 
-  def singleRemoval( self, opFile, targetSE ):
-    """ remove opFile replica from targetSE
+  def removeWithOwnerProxy( self, opFile, targetSE ):
```
Comment (andresailer) [inline DataManagementSystem/Agent/RequestOperations/RemoveReplica.py]: Strictly speaking this is a change of an interface.

Comment (phicharp) [inline DataManagementSystem/Agent/RequestOperations/RemoveReplica.py]: Right, indeed these methods should not be public!!! but this is the case of a large fraction of methods and 90% of data members of our classes... 
I can change that indeed! ;-)

Comment (fstagni) [inline DataManagementSystem/Agent/RequestOperations/RemoveReplica.py]: Maybe just mark it as `_removeWithOwnerProxy`

File: WorkloadManagementSystem/Client/CPUNormalization.py [hunk 2]
```diff
@@ -22,28 +22,53 @@
...
+    totalPower = float( features.get( 'hs06', 0 ) )
+    logCores = float( features.get( 'log_cores', 0 ) )
+    physCores = float( features.get( 'phys_cores', 0 ) )
+    jobSlots = float( features.get( 'jobslots', 0 ) )
+    denom = min( max( logCores, physCores ), jobSlots ) if ( logCores or physCores ) and jobSlots else None
+    if totalPower and denom:
+      return round( totalPower / denom , 2 )
+    else:
+      return None
+  except Exception as e:
```
Comment (fstagni) [inline WorkloadManagementSystem/Client/CPUNormalization.py]: Which Exception can be thrown here? It seems all protected to me.

Comment (phicharp) [inline WorkloadManagementSystem/Client/CPUNormalization.py]: If there is a mistake from the site and the content of the file is not a number... The casting to float() would throw an exception

Comment (fstagni) [inline WorkloadManagementSystem/Client/CPUNormalization.py]: In that you can just catch `ValueError`.

Another point is: if there is a mistake, being here "protected", printout or not, we would end up never noticing and so never notifying the site. I am wondering if we can do better. 

Yet another point, and certainly not for this PR: I don't think any other component that is not the pilot, in DIRAC, use the CPUNormalization. Maybe we can organize this slightly better.

**Comment by chaen** [comment]
Review OK

**Pull request merged**